services/src/main/resources/locale/portlet/news/News_en.properties

```diff
@@ -88,15 +88,15 @@ news.activity.clickToShowDetail=Click here for more news details
 
 news.archive.btn.confirm=Confirm
 news.archive.btn.cancel=Cancel
-news.archive.action=Archive article
+news.archive.action=Bookmark it to easily find it when searching
 news.archive.confirm=This article will be archived and will no longer be available for users. You still can unarchive it from News application.
 news.archive.success=The article was successfully archived
 news.archive.error=An error occurred when trying to archive this article. Try again, if the error persists, please contact your administrator.
 news.archive.label=Archived
 
 news.archive.text=Sorry, you don't have access to this article. Please contact your administrator.
 
-news.unarchive.action=Unarchive article
+news.unarchive.action=Remove from favorites
```

Comment:
Why this value is changed ?

Reply:
As you said preferably that we add another key to avoid changing another label so it must that I add another key and I would change the description of the PR.